autoload/scriptease.vim

```diff
@@ -308,7 +308,11 @@ endfunction
 
 " Section: :Messages
 
-function! scriptease#messages_command(bang) abort
+function! scriptease#messages_command(bang, ...) abort
+  if a:0 && a:1 ==# 'clear'
+    messages clear
+    return
+  endif
```

Comment:
Make this

```vim
if !empty(a:arg)
  return 'messages ' . a:arg
endif
```

This way, the user gets a sensible error if they do something like `:messages clera`.

Also return defaults to `0`, which means the resulting `:execute 0` in the original implementation will jump to the top of the file.

Reply:
done!